web/Controllers/AdminController.cs

```diff
@@ -68,15 +67,49 @@ public IActionResult CancelAdminAccessRequest()
 
         return RedirectToAction(nameof(AdminDashboard));
     }
+    
+    private void LoggerAlreadyAdminWarning(ILogger logger, BricksAndHeartsUser user)
+    {
+        FlashMessage(logger,
+            ($"User {user.Id} already an admin",
+                "danger",
+                "Already an admin"));
+    }
+    
+    [Authorize(Roles = "Admin")]
+    [HttpPost]
+    public ActionResult AcceptAdminRequest(int userToAcceptId)
+    {
+        if (_adminService.GetUserFromId(userToAcceptId) == null)
+        {
+            return View("Error", new ErrorViewModel());

```

Comment:
Not your code again, but tbh I think this should just throw an exception - that will at least log something. I think omit this check completely and let the service method throw an exception (and same for reject)